src/libs/error.js

```diff
@@ -37,6 +37,21 @@ export const reportErrorAndRethrow = _.curry((title, fn) => async (...args) => {
   }
 })
 
+/**
+ *  This function is designed for use in modals
+ *  Modals can overlay any error reporting, with the `throw` in the default `withErrorReporting` preventing the modal itself from closing on error
+ *  As such, we must ensure we call the dismiss function if an error occurs
+ */
```

Comment:
Sorry, I missed a few things on this last time:

```suggestion
/**
 * This function is designed for use in modals.
 * Modals can overlay any error reporting, with the `throw` in the default `withErrorReporting`
 * preventing the modal itself from closing on error.
 * As such, we must ensure we call the dismiss function if an error occurs
 */
```